### DISCUSSION TITLE
Displaying setting

### DISCUSSION THREAD

**Discussion by simonhackler** [bromberry-games/Inkvestigations#169] [UI/UX]
Since we have different settings and timeperiods we need to display them to the users.
This has to be done before the first message. 
We could just write it above the first message.

We can add the title very prominently and then display the setting below.

**Comment by zivky** [comment]
So something like this:
### Title of the mystery
_The year is 1889 and in a little town called bla bla bla_
Introduction letter...

**Reply by simonhackler** [reply]
Yes. Oder when you sherlock suddenly recieve a message. Smth like that.

**Reply by zivky** [reply]
Maybe it would be cool to have a small scene where the murder occurs, but it doesn't say who did it. So the scene would contain both the setting and the murder or something just before the murder. May be a good place to give hints as well.
This should be optional.
